text/0683-deprecate-fallback-resolution-ember-resolver.md

```diff
@@ -0,0 +1,139 @@
+
+The major difference between the two resolvers can be easily linted against:
+
+_It is important to call out that app re-exports are necessary when referencing components coming from an external source or using Wall Street Syntax._
+
+| Lookup                                                  | Ember-resolver                                                                                                                                                            | ember-strict-resolver                                    |
+| ------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------- |
+| service/foo‌ ‌                                          | service:foo‌ ‌                                                                                                                                                            | service:foo‌ ‌                                           |
+| service/foo-bar‌ ‌                                      | service:foo-bar,‌ ‌service:fooBar‌ ‌                                                                                                                                      | service:foo-bar‌ ‌                                       |
+| (external-module-name)/addo‌n/services/foo-bar‌ ‌       | service:(external-module-nam‌e)@foo-bar‌ ‌service:(external-module-name)@fooBar‌                                                                                          | service:(external-module-nam‌e)@foo-bar‌ ‌               |
+| controller/foo/bar‌ ‌                                   | this.controllerFor(‘foo.bar’),‌ ‌this.controllerFor(‘foo/bar’)‌ ‌                                                                                                         | this.controllerFor(‘foo.bar’)‌ ‌                         |
+| component/foo/bar‌ ‌                                    | ‌{{foo/bar}}‌ ‌                                                                                                                                                           | {{foo/bar}}‌ ‌                                           |
+| component/foo/bar-baz‌ ‌                                | ‌{{foo/bar-baz}},‌ ‌{{foo/bar_baz}}‌ ‌                                                                                                                                    | ‌{{foo/bar-baz}}‌ ‌                                      |
+| (external-module-name)/addo‌n/components/foo/bar-baz‌ ‌ | {{component‌ ‌“(external-module-name)/foo/‌bar-baz”}}‌ ‌{{component‌ ‌“(external-module-name)/foo/‌bar_baz”}}‌ ‌{{component‌ ‌“(external-module-name)/foo/‌barBaz”}}‌ ‌ ‌ | {{component‌ ‌“(external-module-name)/foo/‌bar-baz”}}‌ ‌ |
```

Comment:
I don't fully understand this table. What are the three columns trying to show? 

I think it would be best to talk about it from the perspective of the `service:foo` calls (since those are constant from the framework perspective). So for `service:fooBar` (which is what is used when you do `fooBar: injectService()`) the strict resolver would literally look for `app/services/fooBar.js` and the current resolver would look for `app/services/foo-bar.js`.